tests/issues.targets

```diff
@@ -1737,6 +1737,9 @@
         <ExcludeList Include="$(XunitTestBinBase)\tracing\eventsource**">
             <Issue>15494</Issue>
         </ExcludeList>
+        <ExcludeList Include="$(XunitTestBinBase)\tracing\eventactivityidcontrol\eventactivityidcontrol\eventactivityidcontrol.cmd">
```

Comment:
Does this test fail some places? At the end of last week I ended up disabling the failing tests at build because the JIT stress runs were still picking them up for some reason. I need to look into it more still, but targeting the full path like you do here may work. We will just need to run the stress runs and see if this properly excludes the test

Reply:
https://github.com/dotnet/coreclr/issues/15924#issuecomment-360933853

Reply:
I originally disabled this test on Windows because EventPipeEventProvider isn't compiled in on Windows yet.  It occurs to me that we can compile the file in as long as we don't use it in EventSource - this is what breaks things.  So, I'm going to try and compile it in so that this test can run everywhere.  I'll run the JIT stress tests as well.  Thanks for pointing that out.